target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-c2600.dts

```diff
@@ -27,6 +27,12 @@
 
 	chosen {
 		linux,stdout-path = "serial0:115200n8";
+		leds {
+			boot = &power;
+			failsafe = &ledgnr;
+			running = &power;
+			upgrade= &ledgnr;
+		};
 	};
```

Comment:
is this ePAPR conformant ? i thought there was a dedicated section for aliases. also hardcoding this into the dts seems wrong as it puts lede specific software settings into the hardware description file.